src/Curl.Converter.Net/Curl.Converter.Net.csproj

```diff
@@ -0,0 +1,52 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I don't think that the package is not useful per se. I'd suggest merging the code into Curl.Converter.Net project.
Let me know what do you think.

Reply:
Don't really get what you mean. Do you mean merging Curl.Parser.Net to Curl.Converter.Net?

Reply:
> Do you mean merging Curl.Parser.Net to Curl.Converter.Net?

Yes

Reply:
@olsh I think it can be separated because the output from the Curl.Parser.Net can be used to generate for example JS code based on the project need. In this case, they don't have to include the Curl.Converter.Net project. What do you think?